lib/basic_operations.py

```diff
@@ -7,15 +7,30 @@ class BasicOperations:
     @staticmethod
     def add(*args):
         """Adds any number of arguments and returns the result"""
-        for i in args:
-            if type(i) != int:
-                raise Exception
+        # check first how many arguments was passed
+        # if none, return zero
+        # if one, return that
+        # if more than one, get the type of the first one
+        # then use this to check the type of the other arguments
+        string = ""
         if len(args) == 0:
             return 0
         elif len(args) == 1:
```

Comment:
you can change this to `if len(args) == 1` because if len(args) is zero, 0 will be returned and it'll exit the function (wont go here anymore)